File: DoAnOOP/TraSach.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DoAnOOP
{
    public class TraSach
    {
        public ChiTietPhieuMuon chitiet { get; set; }
        public Sach MSSach { get; set; }
        public NhanVien MSNV { get; set; }
        public DateTime ngayTra { get; set; }

        public TraSach(ChiTietPhieuMuon chitiet, Sach MSSach, NhanVien MSNV, DateTime ngayTra)
        {
            this.chitiet = chitiet;
            this.MSSach = MSSach;
            this.MSNV = MSNV;
            this.ngayTra = ngayTra;
        }
        public TraSach(TraSach traSach)
        {
            this.chitiet = traSach.chitiet;
            this.MSSach = traSach.MSSach;
            this.MSNV = traSach.MSNV;
            this.ngayTra = traSach.ngayTra;
        }
    }
}

File: java/cuoiki/src/mypack/TraSach.java
package mypack;

import java.sql.Date;

public class TraSach {
    public ChiTietPhieuMuon chitiet;
    public Sach MSSach;
    public NhanVien MSNV;
    public Date ngayTra;
    
    public TraSach(ChiTietPhieuMuon chitiet, Sach MSSach, NhanVien MSNV, Date ngayTra)
    {
        this.chitiet = chitiet;
        this.MSSach = MSSach;
        this.MSNV = MSNV;
        this.ngayTra = ngayTra;
    }
    public TraSach(TraSach traSach)
    {
        this.chitiet = traSach.chitiet;
        this.MSSach = traSach.MSSach;
        this.MSNV = traSach.MSNV;
        this.ngayTra = traSach.ngayTra;
    }   
    
	public ChiTietPhieuMuon getChitiet() {
		return chitiet;
	}
	public void setChitiet(ChiTietPhieuMuon chitiet) {
		this.chitiet = chitiet;
	}
	public Sach getMSSach() {
		return MSSach;
	}
	public void setMSSach(Sach mSSach) {
		MSSach = mSSach;
	}
	public NhanVien getMSNV() {
		return MSNV;
	}
	public void setMSNV(NhanVien mSNV) {
		MSNV = mSNV;
	}
	public Date getNgayTra() {
		return ngayTra;
	}
	public void setNgayTra(Date ngayTra) {
		this.ngayTra = ngayTra;
	}

    
}
